index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require('functions.php');
+$db = collectionDbConnection();
+$allItems = retrieveData($db);
+$output = displayData($allItems);
+echo $output;
+
+
+
+?>
+
+<!--<!DOCTYPE html>-->
```

Comment:
Commented code generally shouldn't be pushed, it's difficult for other people working on the same repo to know why this was commented out, please uncomment and use php tags to echo $output inside the body